procedural.js

```diff
@@ -0,0 +1,20 @@
+function isPrime(num) {
+	var isPrime = true
+  for (var j=2; j<num/2; j++){
```

Comment:
la condición de parada del `for` debería ser `<=` ya que debe incluir todos los números en el rango.